### DISCUSSION TITLE
Imputation of Longitudinal data

### DISCUSSION THREAD

**Discussion by OscarYan0104yy** [amices/mice#553] [Getting started]
Hi! 

I have been working on imputing my data using the MICE package and have some errors.

The goal is to impute the outcome variable only, which is longitudinal in nature. However there is missingness in the predictor variables as well. Can the package impute my dataset?

I have tried a bunch of different methods specified for longitudinal and multilevel data such as '2l.mer' and so on. Non of them seem to work. 

It would be wonderful if you could provide some suggestions to it!

Thanks a lot!

**Comment by stefvanbuuren** [comment]
Moved to discussion. 

I suggest you first articulate your problem. For example look at <https://stefvanbuuren.name/fimd/ch-longitudinal.html> for possible approaches.

**Reply by OscarYan0104yy** [reply]
Thank you so much for the reply. I've been able to impute the full dataset using tree methods embedded in MICE. Was wondering would this be a suitable method for longitudinal data? Thanks a lot the reply.
